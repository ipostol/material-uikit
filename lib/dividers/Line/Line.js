'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _classnames = require('classnames');

var _classnames2 = _interopRequireDefault(_classnames);

var _Line = require('./Line.scss');

var _Line2 = _interopRequireDefault(_Line);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var Line = function Line(props) {
  return _react2.default.createElement('div', _extends({}, props, { className: (0, _classnames2.default)(_Line2.default.line, props.className) }));
};

exports.default = Line;