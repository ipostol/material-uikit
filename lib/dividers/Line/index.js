'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = undefined;

var _Line = require('./Line');

var _Line2 = _interopRequireDefault(_Line);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.default = _Line2.default;