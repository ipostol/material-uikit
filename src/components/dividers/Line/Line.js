import React from 'react';
import cn from 'classnames';
import styles from './Line.scss';

const Line = props => (
  <div {...props} className={cn(styles.line, props.className)} />
);

export default Line;
