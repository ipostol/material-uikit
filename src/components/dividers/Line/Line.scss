@import '~bidstack-theme/theme';

.line {
  width: 100%;
  height: 1px;
  background-color: $white2;
}
