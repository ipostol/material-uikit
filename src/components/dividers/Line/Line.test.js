import React from 'react';
import { shallow } from 'enzyme';
import Line from './Line';

describe('Line component', () => {

  it('should render correctly', () => {

    const props = {};

    const wrapper = shallow(
      <Line {...props} />,
    );

    expect(wrapper).toMatchSnapshot();

  });

});
