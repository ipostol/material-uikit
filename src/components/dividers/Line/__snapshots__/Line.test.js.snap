// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Line component should render correctly 1`] = `
<div
  className="line"
/>
`;
